menu.rb

```diff
@@ -0,0 +1,21 @@
+# How many menu items would the user like to see?
+puts "Welcome to the crazy random menu generator!"
+puts "How many menu items would you like to create?"
+menu_items = gets.chomp.to_i
+
+# Create 3 arrays, 1. food adjectives, 2. food preparation 3. foods (aka rare animals)
+adj 		= ["Melty", "Spicy", "Buttered", "Gourmet", "Fresh", "Hot", "Crunchy", "Sweet", "Savory", 
+			"Marinated", "Chocolatey", "Flaming", "Creamy", "Polka-dotted", "Chunky"]
+prep 		= ["pan-fried", "sauteed", "steamed", "roasted", "baked", "boiled", "broiled", "fried", 
+			"mashed", "breaded", "scorched", "barbecue", "slow-roasted", "braised", "chopped", "raw", 
+			"sous vide", "poached"]
+rare_animals 	= ["panda", "unicorn", "manatee", "wooly mammoth", "bald eagle", "gorilla", "emperor penguin", 
+			"blue whale", "hammerhead shark", "toucan", "chimpanzee", "hummingbird", "sea turtle", "rhino", 
+			"monarch butterfly", "sloth"]
+
+# Print menu items in an ordered list based on requested number of items by user
+count = 0
+while count < menu_items
+	count += 1
+	puts "#{count}. #{adj.sample} #{prep.sample} #{rare_animals.sample}"
+end
```

Comment:
It is generally a good idea to have a newline character at the end of your file. I know that there is a plugin in atom to do this - I'm guessing there is if you're using sublime text too.